.travis.yml

```diff
@@ -26,6 +26,14 @@ matrix:
       env:
         - MATRIX_EVAL="CXX=c++"
 
+#=============================================================================
+# Cache directories
+#=============================================================================
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/Library/Caches/Homebrew
```

Comment:
note that these changes appear in both PR because I wanted to pass CI failure for this PR